articles/scopes/preview/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Scopes
+description: Overview of scopes.
+toc: true
+---
+
+# Scopes
+
+<div class="alert alert-info">
+<strong>Heads up!</strong> This article describes the latest Auth0 implementation, according to the <a href="/api-auth">API Authorization flows</a>. If you are looking for the legacy Scopes doc refer to <a href="/scopes">Scopes</a>.
+</div>
+
+In OpenID Connect (OIDC) we have the notion of __scopes__, or __claims__. These are strings, sent as part of the `scope` request parameter. These claims can be standard (as defined by the specification) or custom.
```

Comment:
`scope` and `claim` while they look similar they are a lil different. A `claim` is an extended attribute that depending on the token can be said to be related to the user identity (say `email`) or to the users access rights (custom claim in access token).
`scope`s on the other hand refer to reach the client will have when executing on behalf of the user. `scope: openid` is the client asking to retrieve the user's information. `scope:read:todo` is the client trying to read the users TODOs.
In particular, a `scope:email` value is the client asking to retrieve the email of the user that being a standard claim.